openTSNE/affinity.py

```diff
@@ -275,7 +295,7 @@ def build_knn_index(
     data, method, k, metric, metric_params=None, n_jobs=1, random_state=None, verbose=False
 ):
     preferred_approx_method = nearest_neighbors.Annoy
-    if is_package_installed("pynndescent") and sp.issparse(data) and metric not in [
+    if is_package_installed("pynndescent") and (sp.issparse(data) or metric not in [
```

Comment:
This snuck in here somehow.

Reply:
I added it on purpose because I thought otherwise there will be a conflict (as I am modifying the same file in this PR). Not the proper way to handle it?

Reply:
The easiest way would probably be to remove the change here and rebase the branch. And force push here. As it is now, we'll have merge conflicts. As long as you remove this change before you rebase, I think you'll have a clean rebase without any issues.

Reply:
OK removed from here.